plugins/services/unstructured_service.py

```diff
@@ -9,7 +9,8 @@
 from dotenv import load_dotenv
 
 load_dotenv()
-api_key = os.getenv("UNSTRUCTURED_API_URL")
+api_url = os.getenv("UNSTRUCTURED_API_URL")
+api_key = os.getenv("UNSTRUCTURED_API_KEY")
```

Comment:
implementation works, but ideally we should remove both of these .env variables and add them to UserConfig instead. we should add it here also: https://github.com/OriginTrail/edge-node-authentication-service/blob/main/UserConfig.sql

user data is added to the global object in app.py lines 30-35. And it can be accessed through each DAG, as shown on exampleDAG.py lines 31-34. Ideally we should do it this way there and pass those two parameters to the unstructured service/unstructured functions